src/haddock/modules/refinement/emref/__init__.py

```diff
@@ -47,6 +47,12 @@ def _run(self):
             sampling_factor = 1
         if sampling_factor > 100:
             self.log("[Warning] sampling_factor is larger than 100")
+
+        if len(models_to_refine) * sampling_factor > 50000:
```

Comment:
out of scope here, but these numbers should be configurable constants instead of hardcoded

Reply:
should this parameter be in the `general` ones ?